changelog/unreleased/pr-18750.toml

```diff
@@ -0,0 +1,5 @@
+type = "fixed" # One of: a(dded), c(hanged), d(eprecated), r(emoved), f(ixed), s(ecurity)
+message = "Fix not being able to change log level."
```

Comment:
@tellistone @grotlue was this bug actually present in any released version?
Otherwise we should not be releasing a changelog for this.

Reply:
I am not sure if it was present. We can remove the changelog just to be sure.